HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -622,13 +621,13 @@ FastTimerService::PlotsPerPath::book(
   for (unsigned int bin: boost::irange(0u, bins)) {
     auto const& module = job[path.modules_and_dependencies_[bin]];
     std::string const& label = module.scheduled_ ? module.module_.moduleLabel() : module.module_.moduleLabel() + " (unscheduled)";
-    module_counter_          .setBinLabel(bin + 1, label.c_str());
-    module_time_thread_total_.setBinLabel(bin + 1, label.c_str());
-    module_time_real_total_  .setBinLabel(bin + 1, label.c_str());
+    module_counter_          .setBinLabel(bin + 1, label);
+    module_time_thread_total_.setBinLabel(bin + 1, label);
+    module_time_real_total_  .setBinLabel(bin + 1, label);
```

Comment:
Why these changes?

Reply:
This should most likely not be here, but I am not sure where it comes from. Either one of the tools did something crazy or maybe there where commits to this in the last days. Or I screwed up manually. I'll have a look.

(Thanks for looking at this insane list of changes, btw!)

Reply:
Ah, easy. It was the code checks. Not sure what exactly is going on there but apparently our code-checks think the `c_str` is not required (and I'd agree, if it compiles it is probably better like this).